CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ If you are looking for developer-facing changes, check out [CHANGELOG-API.md](./
 
 <!-- next version -->
 
+## v0.106.1
+
+### 🧰 Bug fixes 🧰
```

Comment:
This is the bug fix for the core. Why do we need to make a release for contrib? We only need a release of the contrib binary in the releases repo

Reply:
If someone is building a collector with a contrib component that requires confighttp and doesn't include another component that pulls the 106.1 dependency in. they will run into this issue:

```
2024-07-30T14:44:52.788-0700    info    service@v0.106.0/service.go:117 Setting up own telemetry...
2024-07-30T14:44:52.788-0700    info    service@v0.106.0/service.go:120 OpenCensus bridge is disabled for Collector telemetry and will be removed in a future version, use --feature-gates=-service.disableOpenCensusBridge to re-enable
2024-07-30T14:44:52.788-0700    info    service@v0.106.0/telemetry.go:96        Serving metrics {"address": ":8888", "metrics level": "Normal"}
2024-07-30T14:44:52.788-0700    info    exporter@v0.106.1/builder.go:85 Deprecated component. Will be removed in future releases.       {"kind": "exporter", "data_type": "logs", "name": "loki"}
2024-07-30T14:44:52.788-0700    info    lokiexporter/exporter.go:43     using the new Loki exporter     {"kind": "exporter", "data_type": "logs", "name": "loki"}
2024-07-30T14:44:52.789-0700    info    service@v0.106.0/service.go:199 Starting otelcontribcol...      {"Version": "0.106.0-dev", "NumCPU": 14}
2024-07-30T14:44:52.789-0700    info    extensions/extensions.go:36     Starting extensions...
2024-07-30T14:44:52.789-0700    error   graph/graph.go:432      Failed to start component       {"error": "failed to resolve authenticator \"nop\": authenticator not found", "type": "Exporter", "id": "loki"}
2024-07-30T14:44:52.789-0700    info    service@v0.106.0/service.go:262 Starting shutdown...
2024-07-30T14:44:52.789-0700    info    extensions/extensions.go:63     Stopping extensions...
2024-07-30T14:44:52.789-0700    info    service@v0.106.0/service.go:276 Shutdown complete.
Error: cannot start pipelines: failed to resolve authenticator "nop": authenticator not found
2024/07/30 14:44:52 collector server run finished with error: cannot start pipelines: failed to resolve authenticator "nop": authenticator not found
```

here's the builder config:

```
# NOTE:
# This builder configuration is NOT used to build any official binary.
# To see the builder manifests used for official binaries,
# check https://github.com/open-telemetry/opentelemetry-collector-releases
#
# For the OpenTelemetry Collector Contrib official distribution sources, check
# https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib

dist:
  module: github.com/open-telemetry/opentelemetry-collector-contrib/cmd/otelcontribcol
  name: otelcontribcol
  description: Local OpenTelemetry Collector Contrib binary, testing only.
  version: 0.106.0-dev
  otelcol_version: 0.106.0

extensions:
  - gomod: go.opentelemetry.io/collector/extension/zpagesextension v0.106.0
  - gomod: go.opentelemetry.io/collector/extension/ballastextension v0.106.0

exporters:
  - gomod: go.opentelemetry.io/collector/exporter/nopexporter v0.106.1
  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/lokiexporter v0.106.0

processors:
  - gomod: go.opentelemetry.io/collector/processor/batchprocessor v0.106.0
  - gomod: go.opentelemetry.io/collector/processor/memorylimiterprocessor v0.106.0

receivers:
  - gomod: go.opentelemetry.io/collector/receiver/nopreceiver v0.106.1

connectors:
  - gomod: go.opentelemetry.io/collector/connector/forwardconnector v0.106.1

providers:
  - gomod: go.opentelemetry.io/collector/confmap/provider/envprovider v0.106.1
  - gomod: go.opentelemetry.io/collector/confmap/provider/fileprovider v0.106.1
  - gomod: go.opentelemetry.io/collector/confmap/provider/httpprovider v0.106.1
  - gomod: go.opentelemetry.io/collector/confmap/provider/httpsprovider v0.106.1
  - gomod: go.opentelemetry.io/collector/confmap/provider/yamlprovider v0.106.1
```

It would be possible to workaround the issue for users by adding a replace statement for the import of `confighttp` or adding a v0.106.1 component that pulls the dep in, neither seems great in my opinion

Reply:
ok. makes sense